### ISSUE TITLE
M.2 NVMe SSD - No info displayed

### ISSUE THREAD

**Issue by billgat** [openhardwaremonitor/openhardwaremonitor#1364]

**Comment by yunyuyuan** [comment]
Me too.Hope to add this feature earlier

**Comment by MJAvH** [comment]
On MSI X470 GAMING PRO MAX (MS-7B79) also no info from M.2 NVMe SSD drives.

**Comment by lprocx** [comment]
A little while ago i had the same problem. Fixed it myself in a fork. Haven't had time to follow up on it, but since more people are having problems I might make time for it soon.

Can you you check my fork if it solves your issue? Link: https://github.com/lprocx/openhardwaremonitor

**Comment by donpin1** [comment]
Agree, please add NVMe drives and their temperatures. It would be great if you also had a tab that monitored SMART attributes. Nevertheless, this is easily the best program for me that I have come across. Well done.
![Capture](https://user-images.githubusercontent.com/79115442/108457966-76689580-72d8-11eb-8267-6a8ed24e0f0d.JPG)
